test/helper.js

```diff
@@ -17,24 +20,30 @@ export async function resetGitRepository() {
 }
 
 async function initRepo() {
-  await git.init();
-  git.addConfig('user.name', config.get('history.author').name)
-     .addConfig('user.email', config.get('history.author').email);
+  for (const repoPath of [VERSIONS_PATH, SNAPSHOTS_PATH]) {
```

Comment:
```suggestion
  for (const repoPath of [ VERSIONS_PATH, SNAPSHOTS_PATH ]) {
```

Reply:
I prefer having no spaces between array brackets and other tokens.
Is it important to you and if it is the case can you explain me why please? :)

Reply:
I feel the inner spaces strongly improve readability:

1. It makes it easier to visually distinguish `{`, `[`, `(`.
2. It makes it easier to visually distinguish function calls and boolean groupings from data structures.
3. It unifies the distance between each member of a data structure (otherwise only the innermost members are separated by spaces, not the first and last).

I can live without, but it does really help me to parse the code.

Reply:
What is the English expression to say "Vendu!"? =)

Reply:
“Deal!” 😉

Reply:
Thanks! :) 
So, deal!